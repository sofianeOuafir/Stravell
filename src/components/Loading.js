import React from 'react';

const LoadingPage = ({ size }) => (
  <img className={`loader--${size}`} src="/images/loader.gif" />
);

export default LoadingPage;
