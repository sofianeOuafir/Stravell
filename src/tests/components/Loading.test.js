import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../components/Loading';

test('should correctly render LoadingPage', () => {
  const wrapper = shallow(<Loading />);
  expect(wrapper).toMatchSnapshot();
});
