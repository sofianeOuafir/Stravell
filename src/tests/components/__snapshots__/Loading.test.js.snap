// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render LoadingPage 1`] = `
<img
  className="loader--undefined"
  src="/static/images/loader.gif"
/>
`;
